stats/templates/stats/public_scenario_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base/base.html" %}
```

Comment:
What changes here with respect to the original scenario_list? Otherwise we could just keep that one :-)